pages/2020/index.rst

```diff
@@ -43,6 +43,13 @@ All participants at the conference will be expected to follow the
 More details will be made available on this page closer to the week of the
 conference.
 
+Virtual Hack Day(s)
+###################
+
+In lieu of this year's in-person meeting, which was unfortunately cancelled due to the ongoing COVID-19 pandemic, we will be hosting a 24-hour virtual hack day from 22-24 April. This event is open to anyone who is interested in participating and not limited to those who were planning to attend the in-person meeting. Note that all participants are expected to follow the `code of conduct </code-of-conduct>`_ while participating in any hack day activities.
+
+The hack day will consist of a few talks from some of our invited speakers from the in-person meeting as well as two wrap-up sessions, to accommodate multiple time zones, in which participants can present the results of their hack. Depending on the total number of participants, the SOC may select only a few participants to present their hack day projects. The schedule `can be found here </2020/schedule>`_. More details will be posted in the `Matrix chat room <https://riot.im/app/#/group/+pyastro:openastronomy.org>`_ in the upcoming days.
```

Comment:
We should add the link to the slack team here too if it's still live but I won't block merging this for it, we can add it in a separate PR if necessary.

Reply:
Looks like the Slack invite link is still working so I've pasted it here too.